frontend/src/pages/stats/index.jsx

```diff
@@ -14,33 +15,92 @@ import ActiveAccountsList from "../../components/stats/ActiveAccountsList";
 import ActiveContractsList from "../../components/stats/ActiveContractsList";
 import StakingBar from "../../components/stats/StakingBar";
 
-export default class extends React.Component {
+export default class extends React.PureComponent {
   render() {
+    const chartStyle = {
+      height: "480px",
+      width: "100%",
+      marginTop: "26px",
+      marginLeft: "24px",
+    };
```

Comment:
I wonder if we can use a CSS class instead of passing the object.

Reply:
sorry, I tried but not work